playground/static-site/playwright.config.ts

```diff
@@ -0,0 +1,17 @@
+import type { PlaywrightTestConfig } from "@playwright/test";
+import base from "../playwright.config.js";
+
+/**
+ * See https://playwright.dev/docs/test-configuration.
+ */
+const config: PlaywrightTestConfig = {
+  ...base,
+  /* Run your local dev server before starting the tests */
+  webServer: {
+    command: "npm run dev",
```

Comment:
could we add the `--no-open-browser` flag to this command instead of in the package.json script? that way, if you just want to look at the local dev pages, you don't have to manually open the browser